catalogue_pipeline/merger/src/test/scala/uk/ac/wellcome/platform/merger/MergerFeatureTest.scala

```diff
@@ -41,17 +41,17 @@ class MergerFeatureTest
                     val recorderWorkEntry =
                       createRecorderWorkEntryWith(version = 1)
 
-                    whenReady(storeInVHS(vhs, recorderWorkEntry)) { _ =>
-                      val matcherResult =
-                        matcherResultWith(Set(Set(recorderWorkEntry)))
-                      sendNotificationToSQS(queue, matcherResult)
+                    storeInVHS(vhs, recorderWorkEntry)
```

Comment:
Why removing whenReady? The old logic seemed to call sendNotificationToSQS only when storeInVHS completed successfully. Does it matter? Could this change cause flakyness?

Reply:
The opposite. I didn’t realise I needed the whenReady() and so my test was flaky. I decided to make this function blocking to match our other fixture methods.